nuspec/GregClient.nuspec

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Greg</id>
+    <version>1.0.0</version>
+    <title>Dynamo Package Manager .Net Client</title>
+    <authors>Autodesk</authors>
+    <owners>Autodesk</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>The Dynamo Package Manager .Net Client.</description>
+    <releaseNotes>The Dynamo Package Manager .Net Client.</releaseNotes>
+    <copyright>Copyright 2020</copyright>
+    <license type="expression">MIT</license>
+    <icon>content\images\logo.png</icon>
+    <tags>dynamo package greg</tags>
+    <dependencies>
+      <group>
+        <dependency id="RestSharp" version="105.2.3" />
+        <dependency id="Newtonsoft.Json" version="8.0.3" />
+      </group>
+      <group targetFramework=".NETFramework4.5">
+      </group>
+      <group targetFramework=".NETFramework4.8">
```

Comment:
hmm @alfredo-pozo Why are we depending on both target framework here?

Reply:
Because the actual nuget that is been published, contains both projects, GregClient (4.8) and GregClient45 (4.5) so the definition of the nuspec requires to put all the target frameworks related to the paths on the files section  (e.g. "lib\net45" -> .NetFramework45)

Reply:
@QilongTang the reason we need to multitarget this solution is that we need to deal with TLS 1.0/1.1 deprecation in different ways depending on the .net framework target. The .net 4.5 nugets should be used with older Dynamo 2.0.x releases.

Reply:
@mjkkirschner  @alfarok Thanks for explanation. In this case, I'm good with such change